keyboards/tmo50/keymaps/talljoe/keymap.c

```diff
@@ -0,0 +1,44 @@
+#include "quantum.h"
+#include "talljoe.h"
+
+bool process_indicator_led_user(uint32_t state){
+  return false;
+}
+
+#define LED_MASK (_BV(PB0) | _BV(PB1) | _BV(PB2) | _BV(PB3))
+#define WHITE (_BV(PB0))
+#define YELLOW (_BV(PB1))
+#define MAGENTA (_BV(PB2))
+#define RED (_BV(PB3))
+
+void matrix_scan_keymap(void) {
+  uint32_t lights = WHITE;
+
+  switch(biton32(layer_state))
```

Comment:
This has been aliased now, as we support 8 and 16 bit layer state bitmasks.
```suggestion
  switch(get_highest_layer(layer_state))
```